frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/attachmentUsages.jspf

```diff
@@ -34,7 +34,13 @@
         >
         <thead>
         <tr>
-            <th colspan="7" class="headlabel"><liferay-ui:message key="linked.releases.and.projects" /></th>
+            <th colspan="7" class="headlabel"><liferay-ui:message key="linked.releases.and.projects" />
+            <core_rt:if test="${projectList.size() > 1 or (projectList.size() == 1 and not empty projectList.get(0).linkedReleases)}">
+                (<a href="#" id="expandAll" onclick="$('#AttachmentUsagesInfo').treetable('expandAll');  return false;" class="text-primary"><liferay-ui:message key="expand.all" /> </a>|
+                <a href="#" id="collapseAll" onclick="$('#AttachmentUsagesInfo').treetable('collapseAll'); return false;" class="text-primary"> <liferay-ui:message key="collapse.all" /></a>)
+                <input id="nameSearch" type="input" placeholder="Search" class="float-right">
```

Comment:
As in many places there is language properties for place-holders as well. Kindly update here as well.

Reply:
Changes done.